packages/graph/hash_graph/bin/hash_graph/src/main.rs

```diff
@@ -34,6 +34,7 @@ impl fmt::Display for GraphError {
 /// This will include things that are mocks or stubs to make up for missing pieces of infrastructure
 /// that haven't been created yet.
 async fn stop_gap_setup(pool: &PostgresStorePool<NoTls>) -> Result<(), GraphError> {
+    // TODO: how do we make these URIs compliant
```

Comment:
I'm not really sure what to do about these, and I'm feeling a little burned out from this task now.

Reply:
I assume that we want to remove this function somewhat soon, so I think it's fine to not validate them for now.

Reply:
I'm somewhat inclined to agree but I'm not sure it'll be super soon. I think these will only really be removed when we they are hosted somewhere else, and when we have to handle the logic of storing metadata records about types that live somewhere else (as opposed to creating a new type)

I _think_ it's probably not a big deal, I just worry we'll ignore this for a while and then there'll be some logic that involves crawling through links or exploring definitions and we'll try and resolve these and it'll break.